frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -292,6 +292,7 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                             properties: propertyAllowList?.[TaxonomicFilterGroupType.EventProperties]
                                 ? propertyAllowList[TaxonomicFilterGroupType.EventProperties].join(',')
                                 : undefined,
+                            exclude_hidden: true,
```

Comment:
Unexpected behavior: I saved a property as Hidden from the popover, but it remained in the list, making me unsure if hiding actually works:
<img width="920" alt="Screenshot 2025-02-25 at 22 54 55" src="https://github.com/user-attachments/assets/0e2bc42d-96d6-412c-97b9-0fe294d343d4" />
(it doesn't say "Hidden" for confirmation either)